freqtrade/fiat_convert.py

```diff
@@ -0,0 +1,161 @@
+import logging
+import time
+from pymarketcap import Pymarketcap
+
+logger = logging.getLogger(__name__)
+
+
+class PairConvertion():
+    # Constants
+    CACHE_DURATION = 6 * 60 * 60  # 6 hours
+
+    # Private attributes
+    crypto_symbol = None
+    fiat_symbol = None
+    price = 0.0
+    expiration = 0
+
+    def __init__(self, crypto_symbol: str, fiat_symbol: str, price: float) -> None:
+        """
+        Create an object that will contains the price for a crypto-currency in fiat
+        :param crypto_symbol:
+        :param fiat_symbol:
+        :param price:
+        """
+        self.crypto_symbol = crypto_symbol.upper()
+        self.fiat_symbol = fiat_symbol.upper()
+        self.set_price(price=price)
+
+    def set_price(self, price: float) -> None:
+        """
+        Set the price of the Crypto-currency in FIAT and set the expiration time
+        :param price: Price of the current Crypto currency in the fiat
+        :return: None
+        """
+        self.price = price
+        self.expiration = time.time() + self.CACHE_DURATION
+
+    def is_expired(self) -> bool:
+        """
+        Return if the current price is still valid or needs to be refreshed
+        :return: bool, true the price is expired and needs to be refreshed, false the price is
+         still valid
+        """
+        if self.expiration - time.time() <= 0:
```

Comment:
This could be simplified as `return self.expiration - time.time() <= 0`

Reply:
Done